src/neuroglancer/annotation/renderlayer.ts

```diff
@@ -18,6 +18,9 @@ import 'neuroglancer/annotation/bounding_box';
 import 'neuroglancer/annotation/line';
 import 'neuroglancer/annotation/point';
 import 'neuroglancer/annotation/ellipsoid';
```

Comment:
removing these imports don't affect anything obvious. Still I'd rather add the collections out of caution rather than remove all of them. I'm assuming Jeremy had a reason to do so. The necessity can be explored in a separate project.